llvm/test/tools/llvm-objcopy/MachO/install-name-tool.test

```diff
@@ -0,0 +1,26 @@
+## This test checks general llvm-install-name-tool behavior
+
+# RUN: yaml2obj %s -o %t
+
+## Passing a non-Mach-O binary
+# RUN: not llvm-install-name-tool -add_rpath foo %t 2>&1 | FileCheck %s --check-prefix=NON_MACH_O
+
+# NON_MACH_O: llvm-install-name-tool: error: input file: {{.*}} is not a Mach-O file
```

Comment:
By using FileCheck's -D option, you can replace the `{{.*}}` with a named variable with the exact file name.